### ISSUE TITLE
Unable to differentiate between a nomal TextField and a disabled TextField in high contrast (black) and High Contrast (white) mode when TextField doesn't have any value or placeholder

### ISSUE THREAD

**Issue by SarveshNamjoshi** [microsoft/fluentui#12496]
**Environment Information**
         Package version(s): current
         Browser and OS versions: Microsoft Edge Version 81.0.416.34 (Official build) beta (64-bit)

**Please provide a reproduction of the bug in a codepen:**
https://codepen.io/Namjoshi/pen/YzXgOEx

**Actual behavior:**
In High Contrast (black) and High Contrast (white) modes, one cannot differentiate if the TextField is active or disabled.

**Expected behavior:**
Difference between active and disabled TextField should be easily identified in High Contrast (black) and High Contrast (white) modes.

**Priorities and help requested:**
Requested priority: High

**Comment by aneeshack4** [comment]
![image](https://user-images.githubusercontent.com/4161791/78208291-df44e600-7458-11ea-9caf-1f8375de5e1b.png)
![image](https://user-images.githubusercontent.com/4161791/78208310-e8ce4e00-7458-11ea-900b-08836b8254eb.png)
This doesn't repro ^^. Are you on Edge Chromium? Also this behavior was reported as broken in the past a while ago in #10624 and fixed in PR #11305 already and that code in TextField.styles.ts hasn't had any changes since that PR so this should not have regressed. If it's old Edge specific then that's a browser issue.

**Comment by SarveshNamjoshi** [comment]
@aneeshack4 
1. Yes, I am using **Edge chromium** version Microsoft Edge Version 81.0.416.34 (Official build) beta (64-bit) as I mentioned in the issue earlier as well.
2. Do you mean, placeholder is indicating the difference in normal and disabled TextField?
In my case, I don't have any placeholder to show in the TextField. In this case, it doesn't show any difference in normal and disabled TextFields. Screenshot of my codepen (https://codepen.io/Namjoshi/pen/YzXgOEx) below:

![image](https://user-images.githubusercontent.com/62882875/78239917-ef61d180-74fb-11ea-867c-a7a74a6a444d.png)

**Comment by aneeshack4** [comment]
@SarveshNamjoshi In my screenshot you can see with and without text so I don't this has to do with placeholder.

**Comment by SarveshNamjoshi** [comment]
@aneeshack4,
In your screenshot with high contrast (white), you have put value for disabled TextFieldbut not for active TextField. Also, in high contrast (black) you have value/placeholder for both TextFields.

Whereas, if you refer to my screenshots, my requirement is that I don't have any placeholder or value for the disabled TextField. In that case, the active and disabled TextFields look similar in high contrast mode. 

Could you please refer to my code pen **https://codepen.io/Namjoshi/pen/YzXgOEx**  and try to reproduce the issue.
I am still facing this issue with high contrast.

**Comment by SarveshNamjoshi** [comment]
@aneeshack4, were you able to repro this scenario?

**Comment by SarveshNamjoshi** [comment]
@aneeshack4, were you able to repro this scenario using my code pen?

**Comment by SarveshNamjoshi** [comment]
@joschect / @aneeshack4 
Could you please ry to repro this bug with my code pen https://codepen.io/Namjoshi/pen/YzXgOEx

**Comment by SwapnilMahajan90** [comment]
@joschect  / @aneeshack4  

I am facing similar issue.Could you please consider this on high priority and confirm the behavior by reproducing the above issue.

**Comment by SarveshNamjoshi** [comment]
@joschect / @aneeshack4 
Expecting a confirmation on repro / non-repro of my code pen :  https://codepen.io/Namjoshi/pen/YzXgOEx

**Comment by SrikanthBoyapati** [comment]
Can you please provide an update on this?

**Comment by ecraig12345** [comment]
Sorry for the delay. This works fine in IE/old Edge, but not in Edge Chromium, which means it's mainly due to high contrast issues introduced by the new Edge Chromium (see #11742). 

(Also I noticed that your codepen was using Fabric 6 for some reason. This codepen uses latest Fabric: https://codepen.io/ecraig12345/pen/gOaBOZw )

Here's how that codepen looks in IE/old Edge (which is correct):
![image](https://user-images.githubusercontent.com/5864305/82089080-c3636f80-96a7-11ea-8629-db00151da746.png)

@xugao partially fixed this for Chromium Edge in #11859, but I'm still seeing issues with the disabled state of TextField. So I'm guessing there are some additional overrides we still need to add.
![image](https://user-images.githubusercontent.com/5864305/82089133-dece7a80-96a7-11ea-99d8-7e82bb3d406c.png)

**Comment by ecraig12345** [comment]
Oops turns out the PR fixing this for TextField wasn't merged, but @xugao will work on it.

**Comment by SarveshNamjoshi** [comment]
@ecraig12345, thanks for the updates! any ETA for the fix?

**Comment by xugao** [comment]
@SarveshNamjoshi - sorry for the delay. ETA for fix is by end of week!
